react/features/reactions/functions.any.ts

```diff
@@ -1,42 +1,37 @@
-// @flow
-
 import { v4 as uuidv4 } from 'uuid';
 
 import { getFeatureFlag, REACTIONS_ENABLED } from '../base/flags';
 import { getLocalParticipant } from '../base/participants';
 import { extractFqnFromPath } from '../dynamic-branding/functions.any';
 
-import { REACTIONS, SOUNDS_THRESHOLDS } from './constants';
+import { ReactionEmojiProps, REACTIONS, ReactionThreshold, SOUNDS_THRESHOLDS } from './constants';
 import logger from './logger';
 
 /**
  * Returns the queue of reactions.
  *
  * @param {Object} state - The state of the application.
- * @returns {boolean}
  */
-export function getReactionsQueue(state: Object) {
+export function getReactionsQueue(state: Object): Array<ReactionEmojiProps> {
     return state['features/reactions'].queue;
 }
 
 /**
  * Returns chat message from reactions buffer.
  *
  * @param {Array} buffer - The reactions buffer.
- * @returns {string}
  */
-export function getReactionMessageFromBuffer(buffer: Array<string>) {
+export function getReactionMessageFromBuffer(buffer: Array<string>): string {
     return buffer.map(reaction => REACTIONS[reaction].message).reduce((acc, val) => `${acc}${val}`);
```

Comment:
Could you please add the map return type here as well?

Reply:
Not really needed, because reduce is called after but fine